### PR TITLE
support for locales in static-content deployment

### PR DESCRIPTION
**Pull request by riconeitzel**
you can now `set('static-content-locales', 'en_US de_DE');` in you deploy.php to deploy multiple locales instead of just en_US

| Q             | A
| ------------- | ---
| Bug fix?      | No
| New feature?  | Yes
| BC breaks?    | No
| Deprecations? | No
| Fixed tickets | #2040

> Do not forget to add notes about your changes to CHANGELOG.md. A command line tool has been included to make this easy, see [CONTRIBUTING.md](CONTRIBUTING.md#update-the-changelog) for details.

### CODE REVIEW

File: recipe/magento2.php [hunk 1]
```diff
@@ -4,6 +4,8 @@
 require_once __DIR__ . '/common.php';
 
 // Configuration
+set('static-content-locales', 'en_US');
```
Comment (antonmedv) [inline recipe/magento2.php:7]: Please, use _ underscore in config name.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,7 @@
 [v6.8.0...master](https://github.com/deployphp/deployer/compare/v6.8.0...master)
 
 ### Fixed
+- Recipe for Magento now supports locale configuration for `setup:static-content:deploy`. By default it's using `en_US`. To change that, simply put i. e. `set('static-content-locales', 'en_US de_DE');` in you deployer script. [#2040]
```
Comment (antonmedv) [inline CHANGELOG.md:8]: Usage better put in comment above config option.

Comment (riconeitzel) [reply CHANGELOG.md]: done.

File: recipe/magento2.php (hunk 1 above)
Comment (riconeitzel) [reply recipe/magento2.php]: done.

**Pull request merged**